lib/avs/avs.go

```diff
@@ -85,6 +85,10 @@ func verifyRegisterOperator(ctx context.Context, contracts Contracts, operator c
 		return errors.New("operator not registered with eigen layer delegation manager")
 	}
 
+	// TODO: check if avs at maxOperatorCount
+	// TODO: check if operator has minimum stake
+	// TODO: handle allowlist disabled
```

Comment:
I will handle this - the `handle allowlist disabled` is a bug in current cli for goerli right now. Because our allowlist will be disabled, and operators will fail the `isInAllowlist` check below

Reply:
Addressed https://github.com/omni-network/omni/pull/523